rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ComponentSpecTest.java

```diff
@@ -135,6 +136,10 @@ public void before() throws TException {
                         .setId("1234567890")
                         .setCreatedOn(new SimpleDateFormat("yyyy-MM-dd").format(new Date())));
 
+        when(this.attachmentServiceMock.uploadAttachment(anyObject(), anyObject(), anyObject()))
```

Comment:
Is there a reason for using this pattern, which is different from the existing pattern in line 90? Maybe you could do there (together with the other mock definitions for the attachment service) as well
```
given(this.attachmentServiceMock.uploadAttachment(anyObject(), anyObject(), anyObject())).willReturn(attachment);
```

Reply:
Both will work.I followed the same pattern as it was configured for other post requests like CreateProject. Now I have implemented as suggested.